src/pages/home/report/ReportActionItem.js

```diff
@@ -698,6 +698,13 @@ function ReportActionItem(props) {
         return null;
     }
 
+    // We currently send whispers to all participants and hide them in the UI.
```

Comment:
Does this make things clearer?

```suggestion
    // We currently send whispers to all report participants and hide them in the UI for users that shouldn't see them.
```